pyvista/core/filters/image_data.py

```diff
@@ -1079,6 +1080,13 @@ def contour_labels(  # type: ignore[misc]
                 boundary remains internal even if only one of the two foreground regions
                 on the boundary is selected.
 
+        closed_surface : bool, default: True
+            Generate polygons to "close" the surface at the boundaries of the image.
```

Comment:
```suggestion
            Generate polygons to "pad" the surface at the boundaries of the image.
```

Reply:
```suggestion
            :meth:`Pad <pyvista.ImageDataFilters.pad_image>` the image
            with ``background`` values prior to contouring. This will 
            generate polygons to "close" the surface at the boundaries of the image.
```